File: shoestorefe/lib/domain/entities/order_detail.dart
class OrderDetail {
  final int id;
  final int productId;
  final String? productName;
  final String? productImageUrl;
  final String? color;
  final String? size;
  final int quantity;
  final double unitPrice;

  OrderDetail({
    required this.id,
    required this.productId,
    this.productName,
    this.productImageUrl,
    this.color,
    this.size,
    required this.quantity,
    required this.unitPrice,
  });
}

File: ShoeStore/ShoeStore.Domain/Entities/OrderDetail.cs
namespace ShoeStore.Domain.Entities;

public class OrderDetail
{
    public long Id { get; set; }
    public long OrderId { get; set; }

    public int ProductId { get; set; }
    public int Quantity { get; set; }
    public decimal UnitPrice { get; set; }  // lấy từ StoreProduct.SalePrice tại thời điểm tạo

    public Order Order { get; set; } = null!;
    public Product Product { get; set; } = null!;
}
